### PR TITLE
Fixes for test infrastructure and mongodb support

### PR DESCRIPTION
**Pull request by devkral**
This PR has fixes for travis builds, docs builds, mongodb tests and mongodb support (with djongo).
Travis builds don't fail anymore and there is also a functional test for djongo (mongodb).
The mongodb functional test has a drawback: dj-django-url had to be replaced by plain django settings for the tests:
They are configured in env variables like before but there are more variables.
On the other hand you can add new databases without support in dj-django-url (like djongo).

Sry for the long commit history.

### CODE REVIEW

**Comment by devkral** [comment]
so, every test and task (from mongodb functional to docs build) works now

File: dbbackup/db/base.py [hunk 1]
```diff
@@ -16,7 +16,8 @@
     'django.db.backends.postgresql': 'dbbackup.db.postgresql.PgDumpConnector',
     'django.db.backends.postgresql_psycopg2': 'dbbackup.db.postgresql.PgDumpConnector',
     'django.db.backends.oracle': None,
-    'django_mongodb_engine': 'dbbackup.db.mongo.MongoDumpConnector',
+    'django_mongodb_engine': 'dbbackup.db.mongodb.MongoDumpConnector',
+    'djongo': 'dbbackup.db.mongodb.MongoDumpConnector',
```
Comment (ZuluPro) [inline dbbackup/db/base.py]: djongo ?

Comment (devkral) [reply dbbackup/db/base.py]: it is correct. djongo is a better supported mongo db engine for django

Comment (devkral) [reply dbbackup/db/base.py]: I know strange name ;)

**Comment by devkral** [comment]
ah and this PR adds support for djongo, a django mongodb db engine

**Comment by ZuluPro** [comment]
Hi @devkral 
Sorry for the delay and thank you a lot for your contribution.

**Comment by devkral** [comment]
thanks a lot for this project.

**Pull request merged**